src/main/java/module/transfer/history/PlayerTransferTableModel.java

```diff
@@ -1,89 +1,178 @@
+                            var player = transfer.getPlayerInfo();
+                            if (player != null) {
+                                var age = player.getAgeAtDate(transfer.getDate());
+                                if (age != null) {
+                                    text = age.toString();
+                                    sortValue = age.toDouble();
+                                }
+                            }
+                        }
+                        return new ColorLabelEntry(sortValue, text, ColorLabelEntry.FG_STANDARD, ColorLabelEntry.BG_STANDARD, SwingConstants.LEFT);
+                    }
+                },
+                new TransferTableColumn(id++,"ls.transfer.motherclubfee") {
+                    @Override
+                    public IHOTableEntry getTableEntry(PlayerTransfer transfer) {
+                        return new ColorLabelEntry(CurrencyUtils.convertCurrency(transfer.getMotherClubFee()), ColorLabelEntry.BG_STANDARD, true, 0);
+                    }
+                },
+                new TransferTableColumn(id++,"ls.transfer.previousclubfee") {
+                    @Override
+                    public IHOTableEntry getTableEntry(PlayerTransfer transfer) {
+                        return new ColorLabelEntry(CurrencyUtils.convertCurrency(transfer.getPreviousClubFee()), ColorLabelEntry.BG_STANDARD, true, 0);
+                    }
+                }
+        )).toArray(new TransferTableColumn[0]);
     }
 
-    /** {@inheritDoc} */
-    @Override
-	public final String getColumnName(int column) {
-        return colNames[column];
-    }
-
-    /** {@inheritDoc} */
-    public final int getRowCount() {
-        return values.size();
-    }
-
-    /** {@inheritDoc} */
-	public boolean isCellEditable(int row, int col) {
-        return col == 8;
+    public void setValues(List<PlayerTransfer> values){
+        this.values = values;
+        initData();
     }
 
-    /** {@inheritDoc} */
-    public final Object getValueAt(int rowIndex, int columnIndex) {
-        final PlayerTransfer transfer = values.get(rowIndex);
-
-        return switch (columnIndex) {
-            case 0 -> transfer.getDate().toLocaleDateTime();
-            case 1 -> transfer.getSeason();
-            case 2 -> transfer.getWeek();
-            case 3 -> transfer.getBuyerName();
-            case 4 -> PlayerTransfer.BUY;
-            case 5 -> transfer.getSellerName();
-            case 6 -> transfer.getPrice();
-            case 7 -> transfer.getTsi();
-            case 8 -> transfer.getPlayerInfo()!=null?transfer.getPlayerInfo().getAgeWithDaysAsString(transfer.getDate()):"";
-            case 9 -> new JButton(HOVerwaltung.instance().getLanguageString("ls.button.delete"));
-            default -> ""; //$NON-NLS-1$
-        };
+//
+//    //~ Methods ------------------------------------------------------------------------------------
```

Comment:
Just clean up all the commented out stuff. :-)

Reply:
done